graveler/committed/meta_range_writer.go

```diff
@@ -55,6 +55,7 @@ func (w *GeneralMetaRangeWriter) WriteRecord(record graveler.ValueRecord) error
 		if err != nil {
 			return fmt.Errorf("get range writer: %w", err)
 		}
+		w.rangeWriter.AddMetadata("type", "ranges")
```

Comment:
Since we're going to be reading these back and comparing the value, I think the types should be exported constants

Reply:
Good idea, done!